drone-webdriver/src/test/java/org/jboss/arquillian/drone/webdriver/binary/downloading/source/GitHubSourceAuthenticatedRequestTestCase.java

```diff
@@ -0,0 +1,70 @@
+package org.jboss.arquillian.drone.webdriver.binary.downloading.source;
+
+import io.specto.hoverfly.junit.rule.HoverflyRule;
+import java.util.Base64;
+import java.util.logging.Logger;
+import org.jboss.arquillian.drone.webdriver.utils.GitHubLastUpdateCache;
+import org.jboss.arquillian.drone.webdriver.utils.HttpClient;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.openqa.selenium.remote.DesiredCapabilities;
+
+import static io.specto.hoverfly.junit.core.SimulationSource.classpath;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.ArgumentMatchers.argThat;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+public class GitHubSourceAuthenticatedRequestTestCase {
+
+    private static final Logger log = Logger.getLogger(GitHubSourceAuthenticatedRequestTestCase.class.getName());
+
+    @ClassRule
+    public static HoverflyRule hoverflyRule =
+        HoverflyRule.inSimulationMode(classpath("hoverfly/gh.simulation.mozilla@geckodriver.releases.json"));
```

Comment:
We have `hoverfly-java:0.6.1`, but the latest is `0.8.1` - shall we try to update in this PR?